app/importers/reading/somerville_mega_reading_importer.rb

```diff
@@ -1,67 +1,90 @@
+# For importing reading data from Google Sheets or a local folder of csvs.
 class SomervilleMegaReadingImporter
-  def initialize(options:)
-    @log = options.fetch(:log)
-    @school_local_ids = options.fetch(:school_scope, [])
-    @file_text = options.fetch(:file_text, nil)
-
-    # TODO add dependencies as instance variables
-    # @student_ids_map = ::StudentIdsMap.new
+  def initialize(school_year, options = {})
+    @school_year = school_year
+    @log = options.fetch(:log, Rails.env.test? ? LogHelper::Redirect.instance.file : STDOUT)
     # @matcher = options.fetch(:matcher, ImportMatcher.new)
-    @syncer = ::RecordSyncer.new(log: @log)
-    reset_counters!
+    # @syncer = ::RecordSyncer.new(log: @log)
```

Comment:
There's some leftover commented out code here and in other places.  Let's remove all that before merging.